app/client/src/workers/common/AppComputationCache/index.ts

```
@@ -230,10 +238,8 @@ export class AppComputationCache {
 
       return computationResult;
     } catch (error) {
-      loglevel.error("Error getting cache result:", error);
-      const fallbackResult = await computeFn();
-
-      return fallbackResult;
+      loglevel.error(error);
```

If it can't retrieve it it can recompute the result like what the earlier code is doing.

---

I have moved the fallback logic to where it is consumed. We need to throw the error here so that we can catch it where it is used. This would help us then send the error to the main catch block in eval tree and then eventually send it back to the main thread.